npm/react/examples/visual-sudoku/src/solver/sudoku.js

```diff
@@ -322,6 +321,7 @@ sudoku._search = function(candidates, reverse) {
 
     if (nr_candidates > max_nr_candidates) {
       max_nr_candidates = nr_candidates
+      // eslint-disable-next-line no-unused-vars
```

Comment:
Can we just delete the var if it is unused?

Reply:
Gleb originally had this exception elsewhere, so I didn't want to mess with it.